environment.yml

```diff
@@ -4,7 +4,6 @@ channels:
 dependencies:
   - python=3.10
   - jinja2
-  - pip
-  - pip:
-    - mdutils
-    - pyyaml
+  - pytest-check-links
+  - mdutils
+  - pyyaml
```

Comment:
While I moved these to conda instead of pip, this whole GHA is simple enough to use pure Python tools instead of conda. (This is coming from a heavy conda user! I just prefer to reduce the layers if complexity if possible.)

Reply:
Let me know if you find any issues with the conda packages. One of the hats I wear is of the conda-forge pkg maintainer :-)

Reply:
Works well now! Thanks